VideoApp/VideoApp/Video/Room/ConnectOptionsFactory.swift

```diff
@@ -49,20 +49,8 @@ import TwilioVideo
                     builder.renderDimensions = renderDimensions
                 }
             )
-
-            switch self.appSettingsStore.videoCodec {
-            case .h264:
-                builder.preferredVideoCodecs = [H264Codec()]
-                builder.encodingParameters = EncodingParameters(audioBitrate: 0, videoBitrate: 1_200)
-            case .vp8:
-                builder.preferredVideoCodecs = [Vp8Codec(simulcast: false)]
-                builder.encodingParameters = EncodingParameters(audioBitrate: 0, videoBitrate: 1_200)
-            case .vp8Simulcast:
-                builder.preferredVideoCodecs = [Vp8Codec(simulcast: true)]
-                
-                // Allocate a higher bitrate for the simulcast track with 3 spatial layers
-                builder.encodingParameters = EncodingParameters(audioBitrate: 0, videoBitrate: 1_600)
-            }
+            builder.preferredVideoCodecs = [TwilioVideo.VideoCodec.make(setting: self.appSettingsStore.videoCodec)]
+            builder.encodingParameters = EncodingParameters(audioBitrate: 16, videoBitrate: 0)
```

Comment:
I made assumption it is ok to always use these encoding parameters. Let me know if that is not the case.

Reply:
I think what we agreed to was using a video bitrate of 0 for VGA simulcast, and 16 for audio at all times. I would rather keep the video bitrates since they are specific to capture settings and codecs.